tests/run.sh

```diff
@@ -33,7 +33,7 @@ bazel build tarball smoke_tests
 echo -e "\\e[36m Unpacking artifacts...\\e[0m"
 tar -zxf bazel-bin/tectonic-dev.tar.gz
 cp bazel-bin/tests/smoke/linux_amd64_stripped/go_default_test tectonic-dev/smoke
-export PATH="${PWD}/tectonic-dev/installer:${PATH}"
+alias tectonic="${PWD}/tectonic-dev/installer/tectonic"
```

Comment:
> The alternative would be to store the path in a variable and call it with a $, but that takes away copy and paste compatibility between the script and the terminal, and otherwise has no improvement other than ShellCheck being tricked.

Another alternative is:

```sh
"${PWD}/installer/tectonic" init --config="${CLUSTER_NAME}".yaml
```

and similarly for the other invocations.  You can paste that into a terminal, and it's not tricking ShellCheck.